<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bindDeviceDlg</class>
 <widget class="QDialog" name="bindDeviceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#333333;
}
QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
}
#titleWidget{
background:#ffffff;
border-top-left-radius:4px;
border-top-right-radius:4px;
}
#contentWidget{
background:#eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}
#ptzBtn{
border:1px solid #cccccc;
border-top-left-radius:14px;
border-bottom-left-radius:14px;
border-right:none;
color:#888888;
}
#ptzBtn:checked{
background:#cccccc;
color:#222222;
}
#spyBtn{
border:1px solid #cccccc;
color:#888888;
}
#spyBtn:checked{
background:#cccccc;
color:#222222;
}
#hotBtn{
border:1px solid #cccccc;
border-top-right-radius:14px;
border-bottom-right-radius:14px;
color:#888888;
}
#hotBtn:checked{
background:#cccccc;
color:#222222;
}
#title{
background-color: rgb(255, 255, 255);
border:1px solid #cccccc;
border-bottom:none;
}
#control{
background-color: rgb(255, 255, 255);
border:1px solid #cccccc;
border-top:none;
}
#addBtn{
background:#ffffff;
border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>绑定设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ptzBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>球机</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="spyBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>枪机</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hotBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>热点</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>58</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>120</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="title" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
color:#666666;
}
</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>IP</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>端口</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>操作</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="styleSheet">
              <string notr="true">
#listWidget{
border:1px solid #cccccc;
}

</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="control" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color:#666666;
}
QPushButton:hover{
background:#eeeeee;
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="addBtn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>29</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>29</height>
                 </size>
                </property>
                <property name="text">
                 <string>+ 添加球机</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>点击确定添加热点图片</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#confirmBtn{
background:#ffd500;
border-radius:2px;
color:#333333;
}
#confirmBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#cancelBtn{
background:#ffffff;
border-radius:2px;
color:#333333;
}
#cancelBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">
#clearButton{
background:#ffd500;
border-radius:2px;
color:#333333;
}
#confirmBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>点击清除所有热点</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
